fail2ban/tests/clientreadertestcase.py

```diff
@@ -331,16 +334,34 @@ def testFilterReaderSubstitionDefault(self):
 
 	def testFilterReaderSubstitionSet(self):
 		output = [['set', 'jailname', 'addfailregex', 'to=sour@example.com fromip=<IP>']]
-		filterReader = FilterReader('substition', "jailname", {'honeypot': 'sour@example.com'})
-		filterReader.setBaseDir(TEST_FILES_DIR)
+		filterReader = FilterReader('substition', "jailname", {'honeypot': 'sour@example.com'},
+		  share_config=self.__share_cfg, basedir=TEST_FILES_DIR)
```

Comment:
that is another case for setBaseDir coverage also (look inside it, that branches off 2 times (if self._cfg ... else ...)